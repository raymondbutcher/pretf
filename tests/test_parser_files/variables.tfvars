# https://www.terraform.io/docs/configuration/variables.html

image_id = "ami-abc123"

availability_zone_names = [
  "us-east-1a",
  "us-west-1c",
]

# https://learn.hashicorp.com/terraform/getting-started/variables.html

amis = {
  "us-east-1" = "ami-abc123"
  "us-west-2" = "ami-def456"
}
