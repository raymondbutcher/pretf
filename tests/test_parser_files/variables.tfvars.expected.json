[
    {
        "name": "image_id",
        "value": "ami-abc123"
    },
    {
        "name": "availability_zone_names",
        "value": [
            "us-east-1a",
            "us-west-1c"
        ]
    },
    {
        "name": "amis",
        "value": {
            "us-east-1": "ami-abc123",
            "us-west-2": "ami-def456"
        }
    }
]